.github/labels.yml

```diff
@@ -73,6 +73,10 @@
   description: Removals and Deprecations
   color: 9ae7ea
 
+- name: localization
```

Comment:
Then you need to assign some section in release drafter for it. Probably just create the new one. Worth splitting as a separate PR.